templates/terraform/resource_iam.html.markdown.erb

```diff
@@ -70,6 +70,10 @@ Three different resources help you manage your IAM policy for <%= product.displa
 
 ~> **Note:** `<%= resource_ns_iam -%>_binding` resources **can be** used in conjunction with `<%= resource_ns_iam -%>_member` resources **only if** they do not grant privilege to the same role.
 
+<% unless version == 'ga' || object.iam_policy.iam_conditions_request_type.nil? -%>
```

Comment:
I'm not sure this is quite what we want, as this block is showing up for some IAM resources that don't seem to support conditions:
https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-4252-old..auto-pr-4252#diff-c464ca30afa0357ee26d7b9f783791248ec1a5e35aef490bb6aa062713981e05R34

Reply:
Conditions are supported for IAP.

https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/iap/terraform.yaml

https://github.com/hashicorp/terraform-provider-google/issues/7196